routes/api.php

```diff
@@ -33,3 +34,12 @@
 
 Route::get('/google/redirect', [GoogleAuthController::class, 'redirectToGoogle']);
 Route::get('/google/callback', [GoogleAuthController::class, 'handleGoogleCallback']);
+
+// CourseRegistrations
+Route::middleware('auth:sanctum')->group(function () {
+    Route::controller(CourseRegistrationController::class)->group(function () {
+        Route::get('course-registrations', 'index');
+        Route::post('course-registrations', 'store');
+        Route::put('course-registrations/{courseRegistration}', 'update');
+    });
+});
```

Comment:
zastosuj model binding czyli zrób trasę /courses/{course}/registration

i wtedy we wszyskich trasach będzie widomo do którego kursu się to tyczy, unikniemy podawania go w body
https://laravel.com/docs/11.x/controllers#restful-nested-resources